apis/beacon/attestations/rewards.yaml

```diff
@@ -0,0 +1,37 @@
+get:
+  operationId: getAttestationsRewards
+  summary: Get attestations rewards
+  description: Retrieves rewards info for attestations 
+  tags:
+    - Beacon
+  parameters:
+    - name: epoch
+      in: path
+      required: true
+      description: "The epoch to get rewards info from"
+      schema:
+        $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Uint64"
```

Comment:
It would also be handy to have a query parameter like `?id=0,4,5,6` which can specify a subset of validators for which rewards should be computed. This will save processing time and bandwidth. You can copy the `id` parameter from one of the other endpoints like `getStateValidators`.